features/attached_enable.feature

```diff
@@ -150,6 +150,52 @@ Feature: Enable command behaviour when attached to an UA subscription
            | focal   |
            | xenial  |
 
+    @series.lts
+    Scenario Outline: Attached enable of cis service in a ubuntu machine
+        Given a `<release>` machine with ubuntu-advantage-tools installed
+        When I attach `contract_token` with sudo
+        And I verify that running `ua enable cis` `with sudo` exits `0`
+        Then I will see the following on stdout:
+            """
+            One moment, checking your subscription first
+            Updating package lists
+            Installing CIS Audit packages
+            CIS Audit enabled
+            Visit https://security-certs.docs.ubuntu.com/en/cis to learn how to use CIS
+            """
+        When I run `apt-cache policy usg-cisbenchmark` as non-root
+        Then stdout does not match regexp:
+        """
+        .*Installed: \(none\)
+        """
+        And stdout matches regexp:
+        """
+        \s* 500 https://esm.ubuntu.com/cis/ubuntu <release>/main amd64 Packages
+        """
+        When I run `apt-cache policy usg-common` as non-root
+        Then stdout does not match regexp:
+        """
+        .*Installed: \(none\)
+        """
+        And stdout matches regexp:
+        """
+        \s* 500 https://esm.ubuntu.com/cis/ubuntu <release>/main amd64 Packages
+        """
+        When I verify that running `ua enable cis` `with sudo` exits `1`
+        Then stdout matches regexp
+        """
+        One moment, checking your subscription first
+        CIS Audit is already enabled.
+        See: sudo ua status
+        """
+        And I verify that running `/usr/share/ubuntu-scap-security-guides/cis-hardening/<cis_script> lvl1_workstation` `with sudo` exits `0`
```

Comment:
Lets add a check before hardening:
  And I run `cis-audit level1_server` with sudo
  Then stdout matches regexp
   """
   Title   Ensure root PATH Integrity
   Rule    xccdf_com.ubuntu.xenial.cis_rule_CIS-6.2.6
   Result  pass
   """
   And stdout matches regexp:
    """
   Title   Ensure password expiration is 90 days or less
   Rule    xccdf_com.ubuntu.xenial.cis_rule_CIS-5.4.1.1
   Result  fail
   """

 Might be worth adding also a stdout check re-running cis-audit after your cis-hardening script to ensure that the one "failed" case you audited for was fixed by the harden script